### PR TITLE
Fix fm disk compilation & increase Render1D timeout

### LINKED ISSUE #5237: GH FishboneMoncrief executables don't build on develop
# Bug reports:

### Expected behavior:

They build. (The non-GH ones build fine)

### Current behavior:

For the magnetized case, it complains that the `MagnetizedFmDisk` ID has private member variables because it inheirits privately from `FishboneMoncriefDisk` ID:

```cpp
class MagnetizedFmDisk
    : public virtual evolution::initial_data::InitialData,
      public MarkAsAnalyticData,
      private RelativisticEuler::Solutions::FishboneMoncriefDisk {
```

Making it public causes other issues.

For the non-magnetized case, the `WrappedGr` analytic data can't find a type alias.

### Environment:

clang-10 Debug

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

Fixes #5237 

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by kidder — DISMISSED** [review]

**Review by kidder — DISMISSED** [review]

File: src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ namespace grmhd::AnalyticData {
 class MagnetizedFmDisk
     : public virtual evolution::initial_data::InitialData,
       public MarkAsAnalyticData,
-      private RelativisticEuler::Solutions::FishboneMoncriefDisk {
+      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
```
Comment (knelli2) [inline src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]:
```
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:278:20: error: 'equation_of_state_type' is a protected member of 'RelativisticEuler::Solutions::FishboneMoncriefDisk'

/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp:77:7: note: constrained by protected inheritance here
      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
      InitialData::equation_of_state_type::thermodynamic_dim;
```

**Review by knelli2 — CHANGES_REQUESTED** [review]
The magnetized FM executable still does not build for me

File: src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp (hunk 1 above)
Comment (nilsdeppe) [reply src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]: Compiler, version? It works for me...

Comment (nilsdeppe) [reply src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]: Or more of the error? I don't know, I don't have the error so I can't fix it.

Comment (knelli2) [reply src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]: clang-10 Debug

```
In file included from /home/knelli/Documents/research/spectre/build/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanMagnetizedFmDisk.cpp:6:
In file included from /home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanWithHorizon.hpp:17:
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:278:20: error: 'equation_of_state_type' is a protected member of 'RelativisticEuler::Solutions::FishboneMoncriefDisk'
      InitialData::equation_of_state_type::thermodynamic_dim;
                   ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:419:52: note: in instantiation of static data member 'detail::get_thermodynamic_dim<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false>::value' requested here
      detail::get_thermodynamic_dim<initial_data>::value;
                                                   ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp:424:53: note: in instantiation of static data member 'GhValenciaDivCleanTemplateBase<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>, false, false>::thermodynamic_dim' requested here
      hydro::Tags::EquationOfStateFromOptions<true, thermodynamic_dim>,
                                                    ^
/home/knelli/Documents/research/spectre/src/Evolution/Executables/GrMhd/GhValenciaDivClean/EvolveGhValenciaDivCleanWithHorizon.hpp:55:35: note: in instantiation of template class 'GhValenciaDivCleanTemplateBase<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>, false, false>' requested here
struct EvolutionMetavars : public GhValenciaDivCleanTemplateBase<
                                  ^
/home/knelli/Documents/research/spectre/src/Parallel/Main.hpp:67:35: note: in instantiation of template class 'EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs>' requested here
  using component_list = typename Metavariables::component_list;
                                  ^
/home/knelli/Documents/research/spectre/src/Parallel/CharmMain.tpp:138:9: note: in instantiation of template class 'Parallel::Main<EvolutionMetavars<gh::Solutions::WrappedGr<grmhd::AnalyticData::MagnetizedFmDisk>, false, BondiSachs> >' requested here
  (void)charmxx_main_component{
        ^
/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp:77:7: note: constrained by protected inheritance here
      protected RelativisticEuler::Solutions::FishboneMoncriefDisk {
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/knelli/Documents/research/spectre/src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.hpp:162:9: note: member is declared here
  using equation_of_state_type = EquationsOfState::PolytropicFluid<true>;
```

This is the full error

Comment (nilsdeppe) [reply src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]: Which executable?

Comment (knelli2) [reply src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedFmDisk.hpp]: `EvolveGhValenciaDivCleanMagnetizedFmDisk`

**Review by kidder — DISMISSED** [review]

**Review by knelli2 — APPROVED** [review]
Ok these build now on my machine.

**Pull request merged**